### PR TITLE
blog: add 2023 Q4 project update

### PR DESCRIPTION
**Pull request by notmandatory**

### CODE REVIEW

File: content/authors/daniela-brozzoni/_index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 name: Daniela Brozzoni
 photo:
-twitter: danielabrozzoni
+twitter: daniela-brozzoni
```
Comment (danielabrozzoni) [inline content/authors/daniela-brozzoni/_index.md]: My handle is still `danielabrozzoni`, can you remove this? :)

File: docs/_blog/2024_q4_update.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+November 15: Joined the panel on the [Bitcoin Review Podcast Episode 55](https://bitcoin.review/podcast/episode-55/) to talked about his [padawan-wallet](https://github.com/thunderbiscuit/padawan-wallet) project.
+
+**[Mathew Ramsden](https://github.com/reez/)**
+
+October 11: Spoke at the [Bitcoin Park OpenHouse](https://www.meetup.com/bitcoinpark/events/291768716/) on the topic "Exploring the Lightning Network".
+
+November 8: Created a video for the [Bitcoin Developers](https://www.youtube.com/@bitcoindevelopers) channel on YouTube titled ["Lightning Development with Swift: Make Your First Lightning App with LDK Node Swift"](https://www.youtube.com/watch?v=rcU3LU6iZCs).
+
+**Other current and future contributors...**
```
Comment (danielabrozzoni) [inline docs/_blog/2024_q4_update.md:62]: FYI I asked vlad, but he had no updates

**Review by danielabrozzoni — COMMENTED** [review]
Looks good to me! Thanks for taking care of this ❤️

File: content/authors/daniela-brozzoni/_index.md (hunk 1 above)
Comment (notmandatory) [reply content/authors/daniela-brozzoni/_index.md]: ohh shoot, fixed.

**Review by danielabrozzoni — APPROVED** [review]
ACK 15732633c3881bc02ed64d1a29d6a899d7bdfb77

**Pull request merged**